### ISSUE TITLE
Downloading assets in development mode takes a long time

### ISSUE THREAD

**Issue by manuelbernhardt** [playframework/playframework#903]
Even if only returning 304's, it seems that getting responses for assets does take a long time, about 7 seconds to load all assets.

See e.g. the following:

![assets](https://f.cloud.github.com/assets/459669/298078/a256672a-9551-11e2-8693-5c2dee00a6f8.png)

**Comment by wsargent** [comment]
@manuelbernhardt  Is this issue still happening?

**Comment by manuelbernhardt** [comment]
@wsargent not in any one of the projects (smaller and larger) that I have been working on lately. If `src_managed` is no longer watched then I would presume this is not going to happen any longer, since (in my case) this was the culprit. From an old project's `Build.scala` (Play 2.1): 

```scala
    // temporary workaround for https://github.com/playframework/Play20/issues/903
    // this breaks automatic reloading for changes in the routers but is still better than to reload at each request
    playMonitoredFiles <<= playMonitoredFiles map { (files: Seq[String]) =>
      files.filterNot(file => file.contains("src_managed"))
    }
```

But I haven't needed this in a while now. Plus, since SBT has a new shiny dependency resolution engine, things are much, much (much!) better.

So in conclusion, as far as I am concerned, this is fixed now. Which is great, by the way, because it was driving me insane there for a moment (still getting those nervous ticks when I see Play reloading now and then).

**Comment by mkurz** [comment]
Closing this as fixed then.
Please re-open if the issue reoccurs.
